docs/030_user-guide/100_metrics.md

```diff
@@ -10,6 +10,13 @@ The `MetricsCollector` exposes the following metrics:
 - `pepr_alerts`: A counter that increments when an alert event is triggered in the application.
 - `pepr_Mutate`: A summary that provides the observed durations of mutation events in the application.
 - `pepr_Validate`: A summary that provides the observed durations of validation events in the application.
+- `pepr_cache_miss`: A gauge that provides the number of cache misses per window.
+- `pepr_resync_failure_count`: A gauge that provides the number of failures per resync operation.
```

Comment:
this is based on KFC [resyncFailureMax](https://github.com/defenseunicorns/kubernetes-fluent-client/blob/6a936fd89ed20de0f36c5d7ada96242be84cd92a/src/fluent/watch.ts#L457), we can but i don't know how. What is your suggestion?

Reply:
*resyncFailureCount meant to say

Reply:
"The number of unsuccessful attempts before making the last successful watch connection."

It skips over the word "resync" -- so as not to define something in terms of itself -- and also tries to expand an what the "failure" means -- unsuccessful connection attempts -- so that it's not so open-ended / upsetting.  Something like that maybe?  Or is it missing some nuance?

Reply:
sounds good